<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdtWizardPage1</class>
 <widget class="QWizardPage" name="WdtWizardPage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Save path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="mapBox">
     <property name="placeholderText">
      <string>Azeroth</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Version:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="versionBox">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>Classic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TBC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WotLK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cataclysm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MoP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WoD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Legion</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Map name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="browseBtn">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="pathBox">
     <property name="placeholderText">
      <string>C:/</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browseBtn</sender>
   <signal>clicked()</signal>
   <receiver>WdtWizardPage1</receiver>
   <slot>browsePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browsePath()</slot>
  <slot>textChanged(QString)</slot>
 </slots>
</ui>
